lib/agent0/agent0/hyperdrive/policies/lpandarb.py

```diff
@@ -145,22 +146,36 @@ def apply_step(
 ) -> PoolState:
     """Save a single convergence step into the pool info.
 
-    Arguments
-    ---------
-    pool_state : PoolState
-        The current pool state.
-    bonds_needed : FixedPoint
-        The amount of bonds that is going to be traded.
-    shares_needed : FixedPoint
-        The amount of shares that is going to be traded.
-    gov_fee : FixedPoint
-        The associated governance fee.
+        Arguments
+        ---------
 
 
-    Returns
-    -------
-    pool_state : PoolState
-        The updated pool state.
+
+
+    # TODO: These dataclasses are similar to pypechain except for
```

Comment:
Can you move this TODO elsewhere? It'll mess up the docs generation

Reply:
Actually why is this here at all?

Reply:
i think i had a vim snafu, fixed.